libs/entitlement-runtime/build.gradle

```diff
@@ -11,12 +11,12 @@ apply plugin: 'elasticsearch.publish'
 
 dependencies {
   compileOnly project(':libs:elasticsearch-core')
+  compileOnly project(":libs:elasticsearch-logging")
+  compileOnly project(":libs:elasticsearch-x-content")
 
-  testImplementation project(":test:framework")
-}
-
-tasks.named('forbiddenApisMain').configure {
-  replaceSignatureFiles 'jdk-signatures'
+  testImplementation(project(":test:framework")) {
+    exclude group: 'org.elasticsearch', module: 'elasticsearch-entitlement-runtime'
```

Comment:
Why would `test-framework` bring in the entitlement-runtime project?

Reply:
I didn't realize this project had been moved from libs to distribution. I'm working on fixing everything now when merging with main.

Reply:
Yeah, I think you can just remove this exclude as it's unnecessary.

Reply:
I think this is fixed with the main merge conflicts resolved.